### PR TITLE
[FRENCH] Minor correction

### PR DESCRIPTION
**Pull request by Kcchouette**
-  done on the file `instructions-fr.md.j2`
- done on the file `stunnel-instructions-fr.md.j2`

### CODE REVIEW

**Comment by cpu** [comment]
@Kcchouette Merci! :-)
@alimakki Can you take a :mag: of this PR when you have a chance?

**Comment by alimakki** [comment]
@cpu will do

File: playbooks/roles/openvpn/templates/instructions-fr.md.j2 [hunk 1]
```diff
@@ -19,32 +19,32 @@ Pour des raisons de sécurité, les clients OpenVPN généralement ont leurs pro
...
 1. Cliquez sur *Suivant* et acceptez le contrat de licence en sélectionnant *J'accepte*.
-1. Cliquez *Suivant* sur l'écran *Choose Components* (Choisir les composants). Laissez toutes les options par défaut cochée.
+1. Cliquez *Suivant* sur l'écran *Choose Components* (Choisir les composants). Laissez toutes les options par défaut cochées.
 1. Notez le dossier de destination. C'est là que vous placerez le profil de configuration client `{{ openvpn_direct_profile_filename }}` après l'installation. Cliquez sur *Install* (Installer).
 1. Un avis de sécurité Windows apparaîtra et demande *Voulez-vous installer ce logiciel de périphérique?*. Cliquer *Installer*.
 1. Cliquez *Next* sur l'écran *Installation Complete* (installation complète).
 1. Décochez *Show Readme* puis cliquez *Finish* (finir).
 1. Cliquez-droit sur l'icône OpenVPN GUI et choisissez *Propriétés*.
-1. Allez à l'onglet *Compatibilité* et cliquez sur la coche *Exécuter ce programme en tant qu'administrateur* dans la section *Niveau du privilège*.
+1. Allez à l'onglet *Compatibilité* et cocher la case *Exécuter ce programme en tant qu'administrateur* dans la section *Niveau du privilège*.
```
Comment (alimakki) [inline playbooks/roles/openvpn/templates/instructions-fr.md.j2]: cocher -> cochez

File: playbooks/roles/openvpn/templates/instructions-fr.md.j2 [hunk 2]
```diff
@@ -19,32 +19,32 @@ Pour des raisons de sécurité, les clients OpenVPN généralement ont leurs pro
...
 1. Double-cliquez sur l'icône OpenVPN GUI situé sur votre bureau afin de lancer l'application.
-1. Téléchargez l'un de ces profils OpenVPN unifiés:
+1. Téléchargez l'un de ces profils OpenVPN unifiés :
 {% for client in vpn_client_names.results %}
```
Comment (alimakki) [inline playbooks/roles/openvpn/templates/instructions-fr.md.j2]: l'espace ne semble pas nécessaire:

`Téléchargez l'un de ces profils OpenVPN unifiés :` -> `Téléchargez l'un de ces profils OpenVPN unifiés:`

File: playbooks/roles/openvpn/templates/instructions-fr.md.j2 [hunk 3]
```diff
@@ -19,32 +19,32 @@ Pour des raisons de sécurité, les clients OpenVPN généralement ont leurs pro
...
+1. Ouvrez le répertoire *config* situé dans le dossier de destination. Pour la plupart des utilisateurs, ça sera *C:\Programmes\OpenVPN\config* ou *C:\Program Files (x86)\OpenVPN\config*. Vous verrez un seul fichier README dans ce répertoire.
+1. Faites glisser et déposez le `{{ openvpn_direct_profile_filename }}` téléchargé a ci-dessus, et placez-le dans le même répertoire que le fichier README.
 1. Faites un clic droit sur l'icône OpenVPN dans la barre des tâches et choisissez *Connecter*.
 1. Vous verrez des journaux se déplacent à mesure que la connexion est établie, suivie d'une notification de la barre des tâches indiquant votre adresse IP attribuée.
-1. Succès! Vous pouvez vérifier que votre trafic est correctement routé par [recherche de votre adresse IP sur Google](https://encrypted.google.com/search?hl=fr&q=ip%20address). Il devrait dire *Votre adresse IP publique est {{streisand_ipv4_address}}*.
+1. Succès ! Vous pouvez vérifier que votre trafic est correctement routé par [recherche de votre adresse IP sur Google](https://encrypted.google.com/search?hl=fr&q=ip%20address). Il devrait dire *Votre adresse IP publique est {{streisand_ipv4_address}}*.
```
Comment (alimakki) [inline playbooks/roles/openvpn/templates/instructions-fr.md.j2]: `Succès !` -> `Succès!`

File: playbooks/roles/openvpn/templates/instructions-fr.md.j2 [hunk 4]
```diff
@@ -53,19 +53,19 @@ Pour des raisons de sécurité, les clients OpenVPN généralement ont leurs pro
    * *[Profils alternatifs](#sslh-profiles) sont disponibles si vous êtes sur un réseau qui ne permet l'accès au port `443`.*
    * *[Profils UDP](#udp-profiles) sont disponibles.*
    * *[Profils combinés](#combined-profiles) sont maintenant disponibles.*
-1. Double-cliquez le profile OpenVPN.
+1. Double-cliquez le profil OpenVPN.
 1. Vous devrez choisir si le profil doit être disponible pour tous les utilisateurs ou seulement pour l'utilisateur actuel. Après avoir effectué votre sélection, vous devrez entrer votre mot de passe.
 1. Recherchez l'icône Tunnelblick dans votre barre de menus. Cliquez dessus et choisissez *Connecter*.
-1. Succès! Vous pouvez vérifier que votre trafic est correctement routé par [recherche de votre adresse IP sur Google](https://encrypted.google.com/search?hl=fr&q=ip%20address). Il devrait dire *Votre adresse IP publique est {{streisand_ipv4_address}}*.
+1. Succès ! Vous pouvez vérifier que votre trafic est correctement routé par [recherche de votre adresse IP sur Google](https://encrypted.google.com/search?hl=fr&q=ip%20address). Il devrait dire *Votre adresse IP publique est {{streisand_ipv4_address}}*.
```
Comment (alimakki) [inline playbooks/roles/openvpn/templates/instructions-fr.md.j2]: `Succès !` -> `Succès!`

File: playbooks/roles/openvpn/templates/instructions-fr.md.j2 [hunk 5]
```diff
@@ -53,19 +53,19 @@ Pour des raisons de sécurité, les clients OpenVPN généralement ont leurs pro
...
 <a name="linux"></a>
 ### Linux ###
 1. Installer OpenVPN:
 
    `sudo apt-get install openvpn` où `sudo yum install openvpn` où `gestionnaire-de-paquetage-esoterique hipster openvpn`
 
-   * Si l'installation de OpenVPN via votre gestionnaire de paquetage n'est pas une option, vous pouvez également télécharger et compiler le [code source OpenVPN](/mirror/index-fr.html#openvpn).
-1. Téléchargez l'un de ces profils OpenVPN unifiés:
+   * Si l'installation de OpenVPN via votre gestionnaire de paquet n'est pas une option, vous pouvez également télécharger et compiler le [code source OpenVPN](/mirror/index-fr.html#openvpn).
+1. Téléchargez l'un de ces profils OpenVPN unifiés :
```
Comment (alimakki) [inline playbooks/roles/openvpn/templates/instructions-fr.md.j2]: `unifiés :` -> `unifiés:`

File: playbooks/roles/openvpn/templates/instructions-fr.md.j2 [hunk 6]
```diff
@@ -74,19 +74,19 @@ Pour des raisons de sécurité, les clients OpenVPN généralement ont leurs pro
...
+   * Vous pouvez tester si votre DNS ou non est en train de fuir [ici](https://dnsleaktest.com/).
    * Vous pouvez vous assurer que les serveurs DNS corrects sont utilisés en ajoutant les lignes suivantes au début du fichier `{{ openvpn_direct_profile_filename }}`:
      * `script-security 2`
      * `up /etc/openvpn/update-resolv-conf`
      * `down /etc/openvpn/update-resolv-conf`
 1. Exécutez OpenVPN et passez le profil .ovpn en option.
 
    `sudo openvpn {{ openvpn_direct_profile_filename }}`
-1. Succès! Vous pouvez vérifier que votre trafic est correctement routé par [recherche de votre adresse IP sur Google](https://encrypted.google.com/search?hl=fr&q=ip%20address). Il devrait dire *Votre adresse IP publique est {{streisand_ipv4_address}}*.
+1. Succès ! Vous pouvez vérifier que votre trafic est correctement routé par [recherche de votre adresse IP sur Google](https://encrypted.google.com/search?hl=fr&q=ip%20address). Il devrait dire *Votre adresse IP publique est {{streisand_ipv4_address}}*.
```
Comment (alimakki) [inline playbooks/roles/openvpn/templates/instructions-fr.md.j2]: `Succès !` -> `Succès!`

File: playbooks/roles/openvpn/templates/instructions-fr.md.j2 [hunk 7]
```diff
@@ -131,26 +131,26 @@ Il est préférable de configurer Ubuntu en utilisant le plugin OpenVPN pour Net
    * Cliquez *Valider*.
 1. Cliquez *Enregistrer*
 1. Sélectionnez le VPN dans le menu à gauche, et faites glisser l'interrupteur sur *ON*. Vous pouvez également activer/désactiver le VPN en cliquant sur l'icône WiFi/Réseau dans la barre de menu, défiler vers *Connexions VPN* et en cliquant sur son nom.
-1. Succès! Vous pouvez vérifier que votre trafic est correctement routé par [recherche de votre adresse IP sur Google](https://encrypted.google.com/search?hl=fr&q=ip%20address). Il devrait dire *Votre adresse IP publique est {{streisand_ipv4_address}}*.
+1. Succès ! Vous pouvez vérifier que votre trafic est correctement routé par [recherche de votre adresse IP sur Google](https://encrypted.google.com/search?hl=fr&q=ip%20address). Il devrait dire *Votre adresse IP publique est {{streisand_ipv4_address}}*.
```
Comment (alimakki) [inline playbooks/roles/openvpn/templates/instructions-fr.md.j2]: `Succès !` -> `Succès!`

File: playbooks/roles/openvpn/templates/stunnel-instructions-fr.md.j2 [hunk 8]
```diff
@@ -105,12 +105,12 @@ Maintenant, vous êtes prêt à installer OpenVPN et à le configurer pour achem
 
 #### Installation OpenVPN ####
 
-1. Installer OpenVPN:
+1. Installer OpenVPN :
```
Comment (alimakki) [inline playbooks/roles/openvpn/templates/stunnel-instructions-fr.md.j2]: `OpenVPN :` -> `OpenVPN:`

File: playbooks/roles/openvpn/templates/stunnel-instructions-fr.md.j2 [hunk 9]
```diff
@@ -105,12 +105,12 @@ Maintenant, vous êtes prêt à installer OpenVPN et à le configurer pour achem
...
 
    `sudo apt-get install openvpn` où `sudo yum install openvpn` où `gestionnaire-de-paquetage-esoterique hipster openvpn`
 
    * Si l'installation de OpenVPN via votre gestionnaire de paquetage n'est pas une option, vous pouvez également télécharger et compiler le [code source OpenVPN](/mirror/index-fr.html#openvpn).
-1. Téléchargez l'un de ces profils OpenVPN unifiés:
+1. Téléchargez l'un de ces profils OpenVPN unifiés :
```
Comment (alimakki) [inline playbooks/roles/openvpn/templates/stunnel-instructions-fr.md.j2]: `unifiés :` -> `unifiés:`

File: playbooks/roles/openvpn/templates/stunnel-instructions-fr.md.j2 [hunk 10]
```diff
@@ -126,12 +126,12 @@ Maintenant, vous êtes prêt à installer OpenVPN et à le configurer pour achem
 
    `sudo openvpn {{ openvpn_stunnel_profile_filename }}`
 
-1. Succès! Vous pouvez vérifier que votre trafic est correctement routé par [recherche de votre adresse IP sur Google](https://encrypted.google.com/search?hl=fr&q=ip%20address). Il devrait dire *Votre adresse IP publique est {{streisand_ipv4_address}}*.
+1. Succès ! Vous pouvez vérifier que votre trafic est correctement routé par [recherche de votre adresse IP sur Google](https://encrypted.google.com/search?hl=fr&q=ip%20address). Il devrait dire *Votre adresse IP publique est {{streisand_ipv4_address}}*.
```
Comment (alimakki) [inline playbooks/roles/openvpn/templates/stunnel-instructions-fr.md.j2]: `Succès !` -> `Succès!`

**Comment by Kcchouette** [comment]
For ```Androïde``` → ```Android ```, you can change this on all the project to fix that.
Even if we pronounce ``` /ɑ̃.dʁɔ.id/```, it's a brand and so is written **Android** even on the french wikipedia page.

Four our punctuation, it seems Canadian french doesn't use space; but France french use it (it's why it put some space before the ```!``` and ```:``` mark

File: playbooks/roles/openvpn/templates/instructions-fr.md.j2 (hunk 1 above)
Comment (Kcchouette) [reply playbooks/roles/openvpn/templates/instructions-fr.md.j2]: Fixed!

**Comment by fwhcat** [comment]
guys, make a fr_CA and fr_FR because PR was respecting french in France rules.

**Comment by alimakki** [comment]
@fwhcat having 2 localizations in French seems excessive and would be a lot of duplication for little benefit. @Kcchouette is correct about the spaces after punctuation marks with regards to French (France) grammar rules.

**Pull request merged**